pages/linux/vgchange.md

```diff
@@ -0,0 +1,13 @@
+# vgchange
+
+> Change the attributes of a Logical Volume Manager (LVM) volume group.
+> See also: `lvm`.
+> More information: <https://manned.org/vgchange>.
+
+- Activate or deactivate logical volumes in all volume groups:
```

Comment:
I don't like the usage of the activation or deactivate.

Maybe something similar to `change the status of`

Reply:
@CleanMachine1, how about `change the activation status of`?

Reply:
Thats better. Its fine as it is but I am just being picky

Reply:
@CleanMachine1, I think your phrasing is better because it relates back to the synopsis wording so I've made the change.